README.md

```diff
@@ -9,12 +9,13 @@ James is build with [hoxy](https://github.com/greim/hoxy), [electron](https://gi
 
 ![](resource/screenshot-1.png)
 
-## Installation
+## Running
```

Comment:
Since James isn't really being **installed**, I figured that this would make more sense. Maybe it should be "Developing" instead?